BeerOverflowWindowsApp/App.config

```diff
@@ -8,10 +8,11 @@
     <add key="GoogleAPIKey" value="AIzaSyBqe4VYJPO86ui1aOtmpxapqwI3ET0ZaMY" />
     <add key="GoogleAPILink" value="https://maps.googleapis.com/maps/api/place/nearbysearch/json?location={0},{1}&amp;radius={2}&amp;type={3}&amp;key={4}" />
     <add key="GoogleAPICategories" value="bar,night_club" />
+    <add key ="FourSquareAPILink" value="https://api.foursquare.com/v2/venues/search?client_id={0}&amp;client_secret={1}&amp;ll={2},{3}&amp;categoryId={4}&amp;radius={5}&amp;intent=browse&amp;v=20170901&amp;m=foursquare" />
```

Comment:
Another space

Reply:
We either need spaces everywhere, or no spaces everywhere D:

Reply:
doesnt really make a difference :D

Reply:
STYLE, WE NEED STYLE.
Just kidding. I'll make a commit soon to remove these spaces and fix other requested stuff.